controller/types/types.go

```diff
@@ -15,12 +15,13 @@ import (
 const RouteParentRefPrefix = "controller/apps/"
 
 type ExpandedFormation struct {
-	App       *App                         `json:"app,omitempty"`
-	Release   *Release                     `json:"release,omitempty"`
-	Artifact  *Artifact                    `json:"artifact,omitempty"`
-	Processes map[string]int               `json:"processes,omitempty"`
-	Tags      map[string]map[string]string `json:"tags,omitempty"`
-	UpdatedAt time.Time                    `json:"updated_at,omitempty"`
+	App           *App                         `json:"app,omitempty"`
+	Release       *Release                     `json:"release,omitempty"`
+	ImageArtifact *Artifact                    `json:"artifact,omitempty"`
+	TarArtifacts  []*Artifact                  `json:"tar_artifacts,omitempty"`
```

Comment:
The reason I went with two fields is because most of the time code is concerned with assigning / changing the "image" artifact, it always needs to be present for the release to be run, and there should only be one of them, which I feel is clearer when it has an explicit field rather than appearing in a slice.

WDYT?